src/test/java/hudson/plugins/copyartifact/CopyArtifactWorkflowTest.java

```diff
@@ -146,4 +156,173 @@ private WorkflowJob createWorkflow(String name, String script) throws IOExceptio
         job.setDefinition(new CpsFlowDefinition("node {" + script + "}", true));
         return job;
     }
+
+    // Tests for symbols.
+    // Test only that symbols are accessible as expected.
+    // No tests for features here as they should be tested in other tests.
+    // The latest pipeline supports calling `copyArtifacts` without `step`.
```

Comment:
Please let me know which version introduced the feature to call `SimpleBuildStep`s without `step()`
(I want to update this code comment).

Reply:
Oh, some rather old versions of various plugins. Too old to bother calling out at this point.

Reply:
I used versions used in workflow-cps 2.10.
OK, I'll use the latest versions available for Jenkins-1.642.